lib/web_ui/lib/src/ui/painting.dart

```diff
@@ -1619,6 +1619,15 @@ class ImageFilter {
     //    if (matrix4.length != 16)
     //      throw ArgumentError('"matrix4" must have 16 entries.');
   }
+
+  @override
+  bool operator ==(dynamic other);
+
+  @override
+  int get hashCode;
+
+  @override
+  String toString();
```

Comment:
These all come from `Object` already. Do these blanks have an effect?

Reply:
They allow the api_conform test to pass. I think you are following up elsewhere about fixing those checks in the conform tester, but for now they are needed here or it doesn't pass pre-submit checks.

Reply:
I removed these after the recent change to the api_conform test, but I left the overrides in the subclasses in since they'll be needed if you ever test against the dart tests (and they improve framework updates by allowing ImageFilter objects to be seen as ==).